lbz/configuration/configs.py

```diff
@@ -0,0 +1,58 @@
+from abc import abstractmethod
+from os import getenv
+from typing import Any, Callable, Optional
+
+from lbz.configuration.aws_ssm import SSM
+
+
+class BaseConfig:
+    def __init__(
+        self,
+        key: str,
+        parser: Optional[Callable] = None,
+        default: Optional[Any] = None,
+    ):
+        self.key = key
+        self.parser = parser
+        self.default = default
+        self._value = None
+
+    @abstractmethod
+    def getter(self) -> Any:
+        pass
+
+    def get_value(self) -> Any:
+        if self._value is None:
+            val = self.getter()
+            val = val if val is not None else self.default
+            if val is None:
+                self._value = None
+            else:
+                self._value = self.parser(val) if self.parser else val
+        return self._value
+
+    @property
+    def value(self) -> Any:
+        return self.get_value()
+
+
+class KeyValue(BaseConfig):
```

Comment:
[#961457397](https://github.com/pdyba/lambdalizator/pull/63#discussion_r961457397):
> This can be simply replaced by a globally declared variable - at least I would do that this way to simplify declaring/usage 😉